libs/remix-ui/git/src/components/navigation/branches.tsx

```diff
@@ -20,13 +20,11 @@ export const BranchesNavigation = ({ eventKey, activePanel, callback }) => {
     <>
       <div className={'d-flex justify-content-between pt-1 ' + (activePanel === eventKey? 'bg-light': '')}>
         <span data-id='branches-panel' onClick={()=>handleClick()} role={'button'} className='nav d-flex justify-content-start align-items-center w-75'>
-          {
-            activePanel === eventKey ? <FontAwesomeIcon className='' icon={faCaretDown}></FontAwesomeIcon> : <FontAwesomeIcon className='' icon={faCaretRight}></FontAwesomeIcon>
-          }
-          <label className="pl-1 nav form-check-label">BRANCHES</label>
+          <i className={ activePanel !== eventKey ? "fa fa-caret-right" : "fa fa-caret-down" }></i>
```

Comment:
is this replacing fontawesome icon? Is that necessary. seems unpractical to me to go from typed to untyped. strong chance of typing errors.

Reply:
Reverting